### ISSUE TITLE
Declaration of Zend\Form\Form::bindValues(array $values = Array) must be compatible with Zend\Form\Fieldset::bindValues()

### ISSUE THREAD

**Issue by evolic** [zendframework/zend-form#198]
During my work with Zend Framework tutorial I met folliwing problem:

 - [ ] Declaration of Zend\Form\Form::bindValues(array $values = Array) must be compatible with Zend\Form\Fieldset::bindValues(array $values = Array, ?array $validationGroup = NULL)

### Code to reproduce the issue

Just use the class Zend\Form\Form with PHP 7.2.3

Code works with PHP 7.1.15

### Expected results

Controller should work.

### Actual results

I see white page:
```
[Mon Apr 16 17:30:45 2018] PHP Fatal error:  Declaration of Zend\Form\Form::bindValues(array $values = Array) must be compatible with Zend\Form\Fieldset::bindValues(array $values = Array, ?array $validationGroup = NULL) in /data/projects/web/zf/basket/vendor/zendframework/zend-form/src/Form.php on line 24
[Mon Apr 16 17:30:45 2018] PHP Stack trace:
[Mon Apr 16 17:30:45 2018] PHP   1. {main}() /data/projects/web/zf/basket/public/index.php:0
[Mon Apr 16 17:30:45 2018] PHP   2. Zend\Mvc\Application->run() /data/projects/web/zf/basket/public/index.php:40
[Mon Apr 16 17:30:45 2018] PHP   3. Zend\EventManager\EventManager->triggerEventUntil() /data/projects/web/zf/basket/vendor/zendframework/zend-mvc/src/Application.php:332
[Mon Apr 16 17:30:45 2018] PHP   4. Zend\EventManager\EventManager->triggerListeners() /data/projects/web/zf/basket/vendor/zendframework/zend-eventmanager/src/EventManager.php:179
[Mon Apr 16 17:30:45 2018] PHP   5. Zend\Mvc\DispatchListener->onDispatch() /data/projects/web/zf/basket/vendor/zendframework/zend-eventmanager/src/EventManager.php:322
[Mon Apr 16 17:30:45 2018] PHP   6. Album\Controller\AlbumController->dispatch() /data/projects/web/zf/basket/vendor/zendframework/zend-mvc/src/DispatchListener.php:119
[Mon Apr 16 17:30:45 2018] PHP   7. Zend\EventManager\EventManager->triggerEventUntil() /data/projects/web/zf/basket/vendor/zendframework/zend-mvc/src/Controller/AbstractController.php:105
[Mon Apr 16 17:30:45 2018] PHP   8. Zend\EventManager\EventManager->triggerListeners() /data/projects/web/zf/basket/vendor/zendframework/zend-eventmanager/src/EventManager.php:179
[Mon Apr 16 17:30:45 2018] PHP   9. Album\Controller\AlbumController->onDispatch() /data/projects/web/zf/basket/vendor/zendframework/zend-eventmanager/src/EventManager.php:322
[Mon Apr 16 17:30:45 2018] PHP  10. Album\Controller\AlbumController->addAction() /data/projects/web/zf/basket/vendor/zendframework/zend-mvc/src/Controller/AbstractActionController.php:78
[Mon Apr 16 17:30:45 2018] PHP  11. spl_autoload_call() /data/projects/web/zf/basket/module/Album/src/Controller/AlbumController.php:30
[Mon Apr 16 17:30:45 2018] PHP  12. Composer\Autoload\ClassLoader->loadClass() /data/projects/web/zf/basket/module/Album/src/Controller/AlbumController.php:30
[Mon Apr 16 17:30:45 2018] PHP  13. Composer\Autoload\includeFile() /data/projects/web/zf/basket/vendor/composer/ClassLoader.php:322
[Mon Apr 16 17:30:45 2018] PHP  14. include() /data/projects/web/zf/basket/vendor/composer/ClassLoader.php:444
[Mon Apr 16 17:30:45 2018] PHP  15. spl_autoload_call() /data/projects/web/zf/basket/module/Album/src/Form/AlbumForm.php:6
[Mon Apr 16 17:30:45 2018] PHP  16. Composer\Autoload\ClassLoader->loadClass() /data/projects/web/zf/basket/module/Album/src/Form/AlbumForm.php:6
[Mon Apr 16 17:30:45 2018] PHP  17. Composer\Autoload\includeFile() /data/projects/web/zf/basket/vendor/composer/ClassLoader.php:322
[Mon Apr 16 17:30:45 2018] PHP  18. include() /data/projects/web/zf/basket/vendor/composer/ClassLoader.php:444
```

**Comment by webimpress** [comment]
@evolic  What version of zend-form are you using?

It has been already fixed in https://github.com/zendframework/zend-form/commit/fbda16c3e87725ac64418e8578d167320855f20b and released with 2.11.0 (see: https://github.com/zendframework/zend-form/compare/release-2.10.0...release-2.11.0#diff-2c77116739ece89c91385095c838f093R331).

**Comment by evolic** [comment]
@webimpress It seems that I was using version 2.10.2
Now it is 2.11.0 and all works fine.
